docs/User Interfaces.md

```diff
@@ -16,89 +49,109 @@ Next, create your sub-applications for Nerves and for Phoenix:
 ```bash
 $ cd nervy/apps
 $ mix nerves.new fw
-$ mix phoenix.new ui --no-ecto --no-brunch
+$ mix phx.new ui --no-ecto --no-brunch
 ```
 
-Now, add the Phoenix `ui` app and the `nerves_networking` library to the `fw` app as dependencies:
+Now, add the Phoenix `ui` app and the `nerves_network` library to the `fw` app as dependencies:
 
 ```elixir
-# nervy/apps/fw/mix.exs
+# apps/fw/mix.exs
 
-...
-defp deps do
+# ...
+def deps do
   [{:ui, in_umbrella: true},
-   {:nerves_networking, "~> 0.6.0"}]
+   {:nerves_network, "~> 0.3"}]
 end
-...
+# ...
+```
+
+##### Specifying configuration order
+
+By default, the main config loads the application configs in an unordered way: `import_config "../apps/*/config/config.exs`. This can cause problems if the  `ui` config is applied last: we may lose overrides applied in the `fw` config. You need to force the order in which the config files get imported:
+
+```elixir
+# nervy/config/config.exs
+
+use Mix.Config
+
+import_config "../apps/ui/config/config.exs"
+import_config "../apps/fw/config/config.exs"
 ```
 
-In order to start networking when `fw` boots, add a worker that sets up networking.
-This example sets up the networking using DHCP.
-For more network settings, check the [`nerves_networking` project](https://github.com/nerves-project/nerves_networking).
+
+### Configure networking
+
+In order to start the network when `fw` boots, add `nerves_network` to the `bootloader` configuration in `config.exs`.
 
 ```elixir
-# nervy/apps/fw/lib/fw/application.ex
-
-...
-# add networking
-children = [
-  worker(Task, [fn -> Nerves.Networking.setup :eth0, [mode: "dhcp"] end], restart: :transient)
-]
-...
+# fw/config/config.exs
+
+# ...
+config :bootloader,
+  init: [:nerves_runtime, :nerves_network]
+# ...
 ```
 
-In order to build the `ui` Phoenix application into the nerves `fw` app, you need to add some configuration to your firmware config:
+To set the default networking configuration:
 
 ```elixir
-# nervy/apps/fw/config/config.exs
+# fw/config/config.exs
+
+# ...
+# Use your ISO 3166-1 alpha-2 country code below
```

Comment:
😂

Reply:
I added this in based on the likelihood that users following this guide would be adding in a `wlan0` interface and would miss this detail. But maybe it's enough to just focus on `eth0` and let wireless configuration be handled by the separate docs?

Reply:
Ah yeah, good point. I almost suggested that you simply include the `wlan0` defaults here as well. Maybe that would be more clear, given that there's a problem where `wlan0` doesn't work right if you specify the `ipv4_address_method` option.